README.md

```diff
@@ -27,6 +26,102 @@ In order to contribute, please follow the
 [Contributing Guidelines](
 https://github.com/scality/Guidelines/blob/master/CONTRIBUTING.md).
 
+## Prerequisite
+
+To use vaultclient with an existing S3 Connector deployment, Vault requires
+the use of AWS signature v4 and valid administration credentials
+in its administrative interfaces (that is: to create, delete, and
+list accounts, users, and access keys). To vaultcient use an
+administrative credential (accessKey, secretKey) pair, first create a
+json file like this:
+
+```
+{
+    "accessKey": "<administrative access key for Vault>",
+    "secretKeyValue": "<administrative secret key for Vault>"
+}
+```
+
+For an existing deployment, admin credentials need to be captured from the
+supervisor node. The admin credentials are stored in the same folder as the
+inventory file post S3 Connector deployment completion. For example:
+
+```sh
+$> cat federation/env/s3config/vault/admin-clientprofile/admin1.json
+{
+    "accessKey": "W1JLTJYBDH7XT5YK5SL0",
```

Comment:
since this is documentation, it may be best to stick with
```
    "accessKey": "<administrative access key for Vault>",
    "secretKeyValue": "<administrative secret key for Vault>"
```

for all the places that list accesskey/secret key values

Reply:
I agree with your comment for `README.md` changing administrative credentials to <administrative access...>